content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -422,6 +439,14 @@ However, configuring the proxy this way does require cluster provider specific k
 In addition to that, you loose monitoring of access to external services and cannot apply Istio features on traffic to
 external services.
 
+The third approach directs the traffic through the Istio sidecar proxy, however it allows access to any service on any
+ports that have no HTTP service inside the mesh and no service entry defined. In this approach, similarly to the
+previous one, you cannot monitor access to external services. The advantage of this approach is that you must not
+know which IP ranges are external to the cluster. Additional advantage is that you can decide to switch to the first
```

Comment:
Another advantage is that you can easily switch to the first approach for a specific port by simply creating a service entry for that port.

Reply:
Fixed in https://github.com/istio/istio.io/pull/3222/commits/c2ea3e3a4158a5f9d67868f42c69237b1c08f9f1.